pkg/controller/machineset/controller.go

```diff
@@ -243,7 +243,7 @@ func (c *ReconcileMachineSet) syncReplicas(ms *clusterv1alpha1.MachineSet, machi
 		glog.Infof("Too many replicas for %v %s/%s, need %d, deleting %d", controllerKind, ms.Namespace, ms.Name, *(ms.Spec.Replicas), diff)
 
 		// Choose which Machines to delete.
-		machinesToDelete := getMachinesToDelete(machines, diff)
```

Comment:
This line is the only consumer of `getMachinesToDelete` function so it can be deleted. Doing that please put the:
```
// TODO: Define machines deletion policies.
// see: https://github.com/kubernetes/kube-deploy/issues/625
```
into your implementation for future reference. There is a lot of deletion strategies to choose among. Still worth discussing.

Reply:
make sense. will do.